serverless/serverless
# Built in integration of serverless enterprise

## What did you implement:

PLAT-916



## How did you implement it:

Added SFE plugin as a dependency and automatically load it, explicitly from sls's `node_modules` to avoid issues with global installation.

## How can we verify it:

```
npm i -g serverless/serverless#plat-916
```
if you have logged into SFE, first remove those details from or remove the `~/.serverlessrc` file entirely. Then use sls to do some deploys, you should get the ad about SFE on deploys.

then run `sls login` and deploy again, you should get warnings about configuring app&tenant.

configure app&tenant and deploy again. should now be using SFE!

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
